src/main/scala/temple/generate/language/service/adt/Endpoint.scala

```diff
@@ -3,9 +3,26 @@ package temple.generate.language.service.adt
 sealed trait Endpoint
 
 object Endpoint {
-  case object ReadAll extends Endpoint
-  case object Create  extends Endpoint
-  case object Read    extends Endpoint
-  case object Update  extends Endpoint
-  case object Delete  extends Endpoint
+
+  case object ReadAll extends Endpoint {
```

Comment:
I’d be tempted to refactor this to
```scala
sealed abstract class Endpoint(verb: String)
case object ReadAll extends Endpoint("List")
```
Firstly so it’s less verbose, and secondly because it makes it slightly clearer what the string represents

Reply:
> I’d be tempted to refactor this to
> 
> ```scala
> sealed abstract class Endpoint(verb: String)
> case object ReadAll extends Endpoint("List")
> ```
> 
> Firstly so it’s less verbose, and secondly because it makes it slightly clearer what the string represents

This needs `val` in front of verb to be usable

Reply:
yep, my bad